lib/components/popover.vue

```diff
@@ -103,6 +102,15 @@
                 validator(value) {
                     return value >= 0;
                 }
+            },
+            popoverStyle: {
+                type: Object,
+                default() {
```

Comment:
Right, I was setting it to {} before instead of null, and then I had to return it as a function. This was a leftover of that apparently.